dockerfiles/packaging/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM linagora/james-project:latest as source
+
+FROM debian:8.1
+
+RUN apt-get update
```

Comment:
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#minimize-the-number-of-layers

 - Use && to minimize the number of layers
 - Clean your cache after this `rm -rf /var/lib/apt/lists/\*`

Reply:
what's the point as this image is only locally used?

Reply:
It takes less time to generate it... Anyway, it's not very important IMO